src/vm/fieldmarshaler.cpp

```diff
@@ -3956,14 +3974,15 @@ VOID FieldMarshaler_SafeHandle::UpdateNativeImpl(OBJECTREF* pCLRValue, LPVOID pN
     // A cleanup list MUST be specified in order for us to be able to marshal
     // the SafeHandle.
     if (ppCleanupWorkListOnStack == NULL)
```

Comment:
I would think that this should continue to throw like before. Or is the throw unreachable via public APIs?

Reply:
I think this one is reachable. I'll check, and if it is I'll add back the throw.